tests/RobotFramework/tests/config_management/child_conf_mgmt_plugin.robot

```diff
@@ -74,6 +74,7 @@ Child device config update
 Set external MQTT bind address
     Set Device Context    ${PARENT_SN}
     Execute Command    sudo tedge config set mqtt.external.bind.address ${PARENT_IP}
+    Execute Command    sudo tedge config set http.client.host ${PARENT_IP}
```

Comment:
This change is not backwards-compatible, because the previous behaviour of deriving connect address from bind address was invalid in the first place. As such, the tests were changed, and similar change may be required on users installations as well. The old `http.bind.address` is not deprecated and it remains used, so I'm not sure if it's possible to change both in a backwards-compatible and correct manner.

Reply:
What is the default value of `http.bind.address`? I would assume `127.0.0.1`? If so then at least the existing configuration would still work.